### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/cookbook/vuepress/page.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#875] [Announcements]
# https://theme-hope.vuejs.press/zh/cookbook/vuepress/page.html

VuePress 是以 Markdown 为中心的。你项目中的每一个 Markdown 文件都是一个单独的页面。 路由 默认情况下，页面的路由路径是根据你的 Markdown 文件的相对路径决定的。 假设这是你的 Markdown 文件所处的目录结构: 将 docs 目录作为你的 sourceDir ，例如你在运行 vuepress dev docs ...

https://theme-hope.vuejs.press/zh/cookbook/vuepress/page.html

**Comment by yufile** [comment]
ewqeqw
